db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_05_09_132942) do
+ActiveRecord::Schema[7.0].define(version: 2022_05_13_090750) do
```

Comment:
if we intend not to include the migration can we cleanup the code around adding one please (including the validation in the model and domain model changes etc.)?

Reply:
👍